src/main/java/marquez/api/models/DatasetVersionRequest.java

```diff
@@ -26,7 +27,7 @@
 @JsonSubTypes({@JsonSubTypes.Type(value = DbTableVersionRequest.class, name = "DB")})
 public abstract class DatasetVersionRequest {
   @Getter @NonNull private final DatasetType type;
-  private final String description;
+  @Nullable private final String description;
```

Comment:
this is an important behavior change that is not captured in the description for this PR -- descriptions are now nullable. This PR should probably update the OpenAPI.yaml simultaneously:

https://github.com/MarquezProject/marquez/blob/master/docs/openapi.yml#L337

Reply:
Woops, that's not the right line I linked to, will fix.

Reply:
nvm, looks like it's not actually marked required in the OpenAPI.yaml :)

Reply:
we're good here, but now I'm noticing that the OpenAPI.yaml is a bit behind with required fields. Will file a ticket to review that and make sure we're more consistent across our endpoints.